doc/Quick-Start-Guide.rst

```diff
@@ -118,7 +118,7 @@ Troubleshooting
   try installing packages using pip installer.
   run the following commands if package is ipaddress::
 
-    sudo pip install python-ipaddress
+    sudo pip install ipaddress
```

Comment:
Could you verify with CentOS 7.8?

Reply:
[root@ssc-vm-g3-rhev4-1258 cortx]# pip --version
pip 8.1.2 from /usr/lib/python2.7/site-packages (python 2.7)
[root@ssc-vm-g3-rhev4-1258 cortx]# pip install python-ipaddress
Collecting python-ipaddress
  Could not find a version that satisfies the requirement python-ipaddress (from versions: )
No matching distribution found for python-ipaddress
You are using pip version 8.1.2, however version 21.2.4 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[root@ssc-vm-g3-rhev4-1258 cortx]# pip install ipaddress
Requirement already satisfied (use --upgrade to upgrade): ipaddress in /usr/lib/python2.7/site-packages
You are using pip version 8.1.2, however version 21.2.4 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[root@ssc-vm-g3-rhev4-1258 cortx]# cat /etc/centos-release
CentOS Linux release 7.8.2003 (Core)
[root@ssc-vm-g3-rhev4-1258 cortx]#

Reply:
Changes are good. But add proper comment in commit message.  like
Issue description and about fix.